price-estimator/README.md

```diff
@@ -53,3 +53,31 @@ Example Response:
 
 * `buyAmountInBase` estimates the buy amount (in base tokens) a user can set as a limit order while still expecting to be completely matched when selling the given amount of quote token.
 * The other fields repeat the parameters in the url back.
+
+### Estimated Sell Amount
+
+`GET /api/v1/markets/:market/estimated-sell-amount/:price-in-quote`
+
+* `market` is as above
+* `price_in_quote` is a decimal number.
+
+Url query is as above.
+
+Example Request: `/api/v1/markets/1-7/estimated-sell-amount/245.5?atoms=true`
```

Comment:
Given you are returning both buy and sell amount maybe we should call this route amount-at-price?